src/action_runner/action_runner.vcxproj

```diff
@@ -23,7 +23,7 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <ProjectGuid>{D29DDD63-E2CF-4657-9FD5-2AEDE4257E5D}</ProjectGuid>
     <RootNamespace>actionrunner</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
I might be wrong but I don't think we should be editing this file? My understanding is WindowsTargetPlatformVersion  is determined on every release cycle depending on what windows SDK the lead devs agreed to support

Reply:
I've talked to my coach about this and he said this is normal. If it causes a conflict, it is easy to resolve